package/PKGBUILD

```diff
@@ -40,6 +41,7 @@ sha256sums=(
   "SKIP"
   "SKIP"
   "SKIP"
+  "SKIP"
```

Comment:
generate the SHA256 hash for each source file **excluding the jar one**.
Tips: this is a quick and dirty bash command to print the sha:

```bash
find ./package/* -exec sh -c 'sha256sum "$0"' {} \;
```